aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -109,13 +111,15 @@ def process_message(record: dict, batch_item_failures: dict):
                     '{message.product_code}. {entitlements.error}'
                 )
 
-            request = {  # noqa TODO: Cleanup noqa
+            request_data = {
                 'customerIdentifier': message.customer_id,
                 'marketplaceIdentifier': 'AWS',
                 'productCode': message.product_code,
                 'entitlements': entitlements.get_entitlements()
             }
-            # TODO: Send entitlement update request with entitlement info
+            scc_url = 'https://scc.suse.com/api/cloud/saas/update_entitlement/'
```

Comment:
Do we want to take this information from a service config file ? I know we probably code this only for scc but in theory the endpoint to send this request to could also be something different. Thoughts

Reply:
Yeah, I added another commit with inclusion of config file. I also added the missing requests requirements to spec and toml.